BusinessLogicLayer/Helpers/PaginationFilter.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using BusinessLogicLayer.ViewModels.Pagination;
+
+namespace BusinessLogicLayer.Helpers
+{
+    public class PaginationFilter<T>
+    {
+        public PagedResult<T> GetPagedItems(int pageNumber, int pageSize, List<T> allItems)
```

Comment:
Good enough for current task (because of small amount of tasks) but bad in general.
You may find 10K-100K records in DB.
And taking all them from DB in DAL and then paginating in BLL is not efficient at all.
Please implement a more DB-efficient approach.